airflow/www/package.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
It looks like this file is completely new, I was expecting a 1-2 line change. Is there an alternative way they were managing js dependencies on previous versions of Airflow?

Reply:
In `1.10.3` there are two UI version, one under `airflow/www`, the one we are currently using, and other under `airflow/www_rbac`, containing this latest one the `package.json` with all the versions.
In newest releases, they have unified all under the same place, `airflow/www` so they seem to be sharing the same `package.json`, but in our case an in our version I don't know if a change on `airflow/www_rbac/package.json` would also affect `airflow/www`